auth-api/src/auth_api/models/entity.py

```diff
@@ -29,7 +29,7 @@ class Entity(BaseModel):  # pylint: disable=too-few-public-methods
 
     id = Column(Integer, primary_key=True)
     business_identifier = Column('business_identifier', String(75), unique=True, nullable=False)
-
+    pass_code = Column('pass_code', String(75), unique=False, nullable=True)
```

Comment:
Should we add a constraint here to make sure pass_code's are unique? @pksumesh ?

Reply:
It's not necessary to be unique, as it's generated by some code. We don't know how the code works.

Reply:
That's true.  We'll probably be encrypting passcodes eventually anyway, so it probably doesn't matter.